src/Router.svelte

```diff
@@ -15,10 +15,19 @@
     const path = hashParts[0]
     let query = {}
     if (hashParts[1]) {
-      const parts = decodeURI(hashParts[1])
-        .replace(/"/g, '\\"')
-        .replace(/&/g, '","')
-        .replace(/=/g, '":"')
+      const replacements = [
+        // Remove these because they break our JSON.parse
```

Comment:
How are these getting into the URL? Should they not be encoded by the `qs` library when set?

Reply:
The /t does get an encoded but still broke the JSON.parse here. I added these extra illegal JSON characters just in case even though it's unlikely they end up in the querystring. Either way, not everyone using this package is using qs or encode in the same way so I went for the safe option and added too many checks.

Reply:
I'll merge, you'll still have to ping @coussej to bump the package though